applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -695,15 +695,15 @@ Building and running
 
 .. include:: /includes/ipc_radio_conf.txt
 
-.. include:: ../../../samples/matter/lock/README.rst
-    :start-after: matter_door_lock_sample_nrf70_firmware_patch_start
-    :end-before: matter_door_lock_sample_nrf70_firmware_patch_end
+Building the application on nRF5340 DK with nRF7002EK shield
+============================================================
 
-For example:
+.. include:: /includes/matter_building_nrf5340dk_70ek
```

Comment:
This should be .rst as well?

Reply:
For all other includes, there are .txt files.